### ISSUE TITLE
Cannot hard code duration in events.tsv

### ISSUE THREAD

**Issue by celprov** [TheAxonLab/hcph-sops#261]
We cannot hard code the duration of the sub-tasks in the `events.tsv`, because we have changed the task design over time.
E.g in our current design the breath hold lasts 25s, while in previous design it used to last only 15s.

Thus the `write_event_file.py` needs to be adapted.

**Comment by oesteban** [comment]
> E.g in our current design the breath hold lasts 25s, while in previous design it used to last only 15s.

O_O this should not be the case, in particular.

As I discussed with Elodie, we want to generate the events files with three different approaches:
- From psychopy's log (potentially of interest going forward: the psychopy-bids plugin)
- From the AcqKnowledge file (from triggers)
- From the ET log (from messages sent to the ET)

We then want to check the alignment of these three and make sure we get to the same thing.

**Comment by esavary** [comment]
The breath hold lasts for 15 seconds, but when we were revising the task together a few months ago, we included a 10-second period with a rectangle of the same color as the background immediately following the breath hold (essentially a 10-second break at the end of each breath hold block). I reviewed the manuscript, and this break isn't mentioned. Should I remove it? (We decided this at the same time we changed the color for the last second of the breath-hold, but I don't remember exactly why we made this choice.)
